### ISSUE TITLE
Rojo source map generation

### ISSUE THREAD

**Issue by SecondNewtonLaw** [JohnnyMorganz/luau-lsp#709]
While working on a Rojo project with several project files `*.project.json`. Luau-Lsp will choose `default.project.json` ALWAYS to generate the source map, even when regenerating it from the contextual menu.
Normally, this would be ok, but when running another project file that is not it problems arise. The source map will not match what is actually in the DataModel, rather it will match the DataModel of `default.project.json`. If the difference between project files is significant, like the `Packages` being on different places, it will cause Luau-Lsp think you're doing errors, while you are not.

tl;dr: When generating a source map, no matter if it is from the contextual menu, it will always choose to use `default.project.json` which causes problems when you have multiple projects and the structure they use is different enough

**Comment by JohnnyMorganz** [comment]
We have a project file setting you can configure to change which project.json the sourcemap is created from. Does that work for you?

**Comment by SecondNewtonLaw** [comment]
Oh dang it, alright I see it, I completely forgot about the setting, completely flies under the radar, ty
